lib/publiccloud/instance.pm

```diff
@@ -401,6 +415,24 @@ sub wait_for_ssh {
     return;
 }
 
+sub journal_upload {
+    # moved this code from wait_for_ssh into THIS temporary function
+    # for faster log provisioning and code reuse, improvement and control.
+    my ($self, $log, %args) = @_;
+    my $usr = $args{username};
+    my $com = 'sudo journalctl -b --no-pager > ' . $log;
+    eval {
+        return unless ($log);
+        if (!get_var('PUBLIC_CLOUD_SLES4SAP')) {
+            $self->run_ssh_command(cmd => $com, proceed_on_failure => 1, username => $usr, timeout => 360);
+            $self->upload_log($log, failok => 1);
+        }
+    };
+    if ($@) {
+        print "ERR: $@\n";
```

Comment:
there are native openQA's ways to log something into openQA logs please use them instead print 
```suggestion
        bmwqemu::diag("ERR: $@\n");
```

Reply:
ok thank you, wiil update it